senders/telegram/send_test.go

```diff
@@ -168,3 +170,50 @@ func TestPrepareAlbum(t *testing.T) {
 		})
 	})
 }
+
+func TestCheckBrokenContactError(t *testing.T) {
+	logger, _ := logging.ConfigureLog("stdout", "debug", "test", true)
```

Comment:
```suggestion
	logger, _ := logging.ConfigureLog("stdout", "info", "test", true)
```

not to clutter the tests output

Reply:
Updated